vignettes/mmrm_review_methods.Rmd

```diff
@@ -8,93 +32,93 @@ The FEV data set
 tableone::tableone(fev_data) # something like this to illustrate fev_data
 ```
 
-## Models
+# Models
 
-### Unstructured Covariance
+## Unstructured Covariance
 
-#### SAS
+### SAS
 ```{sas, eval=FALSE}
 PROC MIXED DATA = fev_data cl method=reml;
 CLASS RACE(ref = 'Asian') AVISIT(ref = 'VIS1') SEX(ref = 'Male') ARMCD(ref = 'PBO') USUBJID;
 MODEL FEV1 = ARMCD AVISIT ARMCD*AVISIT RACE SEX / ddfm=satterthwaite solution chisq;
 REPEATED AVISIT / subject=USUBJID type=un r rcorr;
 ```
-#### mmrm
+### mmrm
 ```{r, eval=FALSE}
 mmrm::mmrm(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID),
   data = fev_data)
 ```
-#### nlme
+### nlme
 ```{r, eval = FALSE}
 nlme::gls(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT,
           data = fev_data,
           correlation = nlme::corSymm(form = ~AVISIT | USUBJID),
           weights = nlme::varIdent(form = ~1|AVISIT),
           na.action = na.omit)
 ```
-#### lme4
 
-#### glmmtmb
+### lme4
 
-### Unstructured Covariance with Weights
 
-#### SAS
+### glmmtmb
+
+## Unstructured Covariance with Weights
+
+### SAS
 ```{sas, eval=FALSE}
 PROC MIXED DATA = fev_data cl method=reml;
 CLASS RACE(ref = 'Asian') AVISIT(ref = 'VIS4') SEX(ref = 'Male') ARMCD(ref = 'PBO') USUBJID;
 MODEL FEV1 = RACE SEX ARMCD AVISIT ARMCD*AVISIT / ddfm=satterthwaite solution chisq;
 REPEATED AVISIT / subject=USUBJID type=un r rcorr;
 WEIGHT WEIGHT;
 ```
-#### mmrm
+### mmrm
 ```{r, eval=FALSE} 
 mmrm::mmrm(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID),
   data = fev_data,
   weights = WEIGHT)
 ```
 
-### Toeplitz (heterogeneous)
+## Toeplitz (heterogeneous)
 
-#### SAS
+### SAS
 ```{sas, eval=FALSE}
 PROC MIXED DATA = fev_data cl method=ml;
 CLASS AVISIT(ref = 'VIS4') USUBJID;
 MODEL FEV1 = / ddfm=satterthwaite solution chisq;
 REPEATED AVISIT / subject=USUBJID type=TOEPH r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr;
 ```
 
-#### mmrm
+### mmrm
 ```{r, eval=FALSE}
 mmrm::mmrm(formula = FEV1 ~ toeph(AVISIT | USUBJID),
   data = fev_data,
   reml = FALSE)
 ```
 
-### Toeplitz (grouped heterogeneous)
-#### SAS
-```{sas, eval=FALSE}
-PROC MIXED DATA = ana.dat cl method=ml;
+## Toeplitz (grouped heterogeneous)
+### SAS
+<pre><code>PROC MIXED DATA = ana.dat cl method=ml;
 CLASS AVISIT(ref = 'VIS4') USUBJID ARMCD;
 MODEL FEV1 = / ddfm=satterthwaite solution chisq;
-REPEATED AVISIT / subject=USUBJID type=TOEPH r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr group=ARMCD;
-```
+REPEATED AVISIT / subject=USUBJID <b>type=TOEPH</b> r=1, 2, 3, 4, 5, 6, 7, 8, 9, 10 rcorr <b>group=ARMCD</b>;
+</code></pre>
```

Comment:
@danielinteractive, I know that auto-highlighted code text is nice, but what do you think about highlighting the important bits like this?

<img width="867" alt="Screenshot 2022-11-14 at 20 09 58" src="https://user-images.githubusercontent.com/5337833/201745390-ef46911f-c282-49cd-a341-062ded57f165.png">

Reply:
looks good! good idea